node/parallel/src/service.rs

```diff
@@ -155,24 +155,16 @@ pub fn new_partial(
 ///
 /// This is the actual implementation that is abstract over the executor and the runtime api.
 #[sc_tracing::logging::prefix_logs_with("Parachain")]
-async fn start_node_impl<RB>(
+async fn start_node_impl(
```

Comment:
if it's ok to change this method signature, could you introduce more details about it?

Reply:
Do you mean if it's ok to remove the 'rpc_ext_builder' parameter of start_node_impl?
I think it is only a choice to create rpc_ext_builder before or in the start_node_impl function.
But the creation of rpc_ext_builder depends on the transaction pool which is created in the start_node_impl function, so removed the parameter of rpc_ext_builder directly.